erp_facade_service_openApi.yml

```diff
@@ -0,0 +1,210 @@
+openapi: 3.0.1
+info:
+  title: UKHO ERP Facade Service API
+  description: |
+     UKHO ERP Facade Service shall process S57 and S100 Content publish event received from Enterprise Event Service.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
+  version: 0.0.1
+servers:
+  - url: https://erpfacade-dev-api.azurewebsites.net
+    description: Dev URL
+paths:
+  /api/v2/webhook:
+    options:
+      summary: "Handle CloudEvent webhook validation"
+      tags:
+        - "ERP Facade"
+      description: "Handles CloudEvent webhook validation request."
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorised - either you have not provided valid token, or your token is not recognised.
+        "403":
+          description: Forbidden - you have no permission to use this API  
+        "500":
+          description: Internal Server Error.
+    post:
+      summary: "Webhook to receive events from Enterprise Event Service."
+      security:
+        - jwtBearerAuth: []
+      tags:
+        - "ERP Facade"
+      operationId: "eesWebhook"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/eventSchema'
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorised - either you have not provided valid token, or your token is not recognised.
+        "403":
+          description: Forbidden - you have no permission to use this API
+        "500":
+          description: Internal Server Error.
+  /api/v2/callback/sap/s100actions/processed:
+    post:
+      tags:
+        - "ERP Facade"
+      summary: Returns response status for the callback request from SAP.
```

Comment:
```suggestion
summary: Callback endpoint for SAP to report completion of processing.
```

Reply:
Updated as suggested in different PR https://github.com/UKHO/erp-facade/pull/239. Thanks